packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/container.tsx

```diff
@@ -135,6 +137,18 @@ const SignupConfirmCodeContainer = ({
     return <OAuthDataError error={oAuthDataError} gleanMetric={GleanMetrics.signupConfirmation.error} />;
   }
   if (oAuthKeysCheckError) {
+    if (!keyFetchToken || !unwrapBKey) {
+      const localizedErrorMessage = ftlMsg.getMsg(
+        'signin-code-expired-error',
+        'Code expired. Please login again.'
+      )
+      navigateWithQuery('/signin', {
```

Comment:
Just to confirm: this does not re-open https://mozilla-hub.atlassian.net/browse/FXA-10617 from the browser crash scenario and the user is signed in to the browser/Sync successfully? I know it fixes it for the refresh scenario (🙌).

Reply:
I tested this flow and it opens the bug above. In my opinion, this is might be the lesser of two evils. Ideally, we should request the login link (with new params) and start the flow over. Filed https://mozilla-hub.atlassian.net/browse/FXA-11915 for that.

I think it might be worthwhile to try this approach and see if/how much it improves our conversion.